src/mjolnir/pbfgraphparser.cc

```diff
@@ -78,17 +78,30 @@ struct graph_callback : public OSMPBF::Callback {
 
   virtual void
   node_callback(uint64_t osmid, double lng, double lat, const OSMPBF::Tags& tags) override {
-    // Check if it is in the list of nodes used by ways
-    if (!shape_.get(osmid)) {
-      return;
-    }
```

Comment:
sadly this will slow down parsing the planet because it means we need to look at all nodes' tags before we can bail. is it possible to put this `if` back but to extend it so that its like

    IF we arent processing bss AND we dont care about this OSMID THEN:
        SKIP

im not sure if we know in the code a tthis point in time whether or not we are interested in parsing bss

Reply:
> im not sure if we know in the code a tthis point in time whether or not we are interested in parsing bss

the answer is Yes. :)